projects/eventonica-project.md

```diff
@@ -100,20 +115,34 @@ We'll get to that in a future week.
 [sqle]: https://sqlectron.github.io/#gui
 [nm]: https://nodemon.io/
 
-## Lab Instructions 
+
+## Lab Instructions
 
 ### Starter Code
 
 You'll be making your own fork of this [starter code](https://github.com/Techtonica/curriculum/projects/Eventful-API-test-page), and building on top of it.
 
-### Lab Exercise 
+### Lab Exercise
 
 **Note:** In this project, the instructions to complete the project after the
 installation & setup steps will be increasingly vague. This is done
 intentionally to get you used to thinking like a software engineer.
 
 #### Part 1 - Installation and Setup
 
+
+1. Make sure Sequel Pro is installed properly by creating a test table and query all of its entries
+2. Each group forks the starter code, and then every apprentice clones the forked version to her computer
```

Comment:
Why is this here?

Reply:
> But why are these files here? The issue is for prereq sections.

I linked sections in **Languages** section. I have not modified other lines. I thought it would be easy for someone who is learning this lesson. 😊

Reply:
That’s great, but there’s a lot of new tutorial code here, too. Please fix this branch or make a new one with only changes for this issue. Google it or ask on the slack git channel if you’re not sure how.

Reply:
This is the wrong version; put your changes in the master version, please.